src/azure-cli/azure/cli/command_modules/signalr/tests/latest/test_signalr_private_endpoint.py

```diff
@@ -0,0 +1,100 @@
+        sku = 'Standard_S1'
+        unit_count = 1
+        location = 'eastus'
+
+        self.kwargs.update({
+            'location': location,
+            'signalr_name': signalr_name,
+            'sku': sku,
+            'unit_count': unit_count,
+            'vnet': 'vnet1',
+            'subnet': 'subnet1',
+            'private_endpoint': 'private_endpoint1',
+            'private_endpoint_connection': 'private_endpoint_connection1'
+        })
+
+        signalr = self.cmd('az signalr create -n {signalr_name} -g {rg} --sku {sku}  -l {location}', checks=[
+            self.check('name', '{signalr_name}'),
+            self.check('location', '{location}'),
+            self.check('provisioningState', 'Succeeded'),
+            self.check('sku.name', '{sku}')
+        ]).get_output_in_json()
+
+        # Prepare network
+        self.cmd('network vnet create -g {rg} -n {vnet} -l {location} --subnet-name {subnet}')
+        self.cmd('network vnet subnet update --name {subnet} --resource-group {rg} --vnet-name {vnet} --disable-private-endpoint-network-policies true')
+
+        self.kwargs.update({
+            'signalr_id': signalr['id']
+        })
+
+        # Create a private endpoint connection
+        self.cmd('network private-endpoint create --resource-group {rg} --vnet-name {vnet} --subnet {subnet} --name {private_endpoint}  --private-connection-resource-id {signalr_id} --group-ids signalr --connection-name {private_endpoint_connection} --location {location} --manual-request')
+
+        # Test private link resource list
+        self.cmd('network private-link-resource list -n {signalr_name} -g {rg} --type Microsoft.SignalRService/signalr', checks=[
+            self.check('length(@)', 1)
+        ])
+
+        s_r = self.cmd('signalr show -n {signalr_name} -g {rg}').get_output_in_json()
+        self.kwargs.update({
+            'private_endpoint_connection_id': s_r['privateEndpointConnections'][0]['id']
+        })
+
+        # Test show private endpoint connection
+        self.cmd('network private-endpoint-connection show --id {private_endpoint_connection_id}', checks=[
+            self.check('id', '{private_endpoint_connection_id}'),
+            self.check('properties.privateLinkServiceConnectionState.status', 'Pending')
+        ])
+
+        # Test list private endpoint connection
+        self.cmd('network private-endpoint-connection list --id {signalr_id}', checks=[
+            self.check('length(@)', 1)
+        ])
+
+        # Test approve private endpoint connection
+        self.cmd('network private-endpoint-connection approve --id {private_endpoint_connection_id}', checks=[
+            self.check('id', '{private_endpoint_connection_id}'),
+            self.check('properties.privateLinkServiceConnectionState.status', 'Approved')
+        ])
+
+        # Test reject private endpoint connection
+        self.cmd('network private-endpoint-connection reject --id {private_endpoint_connection_id}', checks=[
```

Comment:
Can we add one more test for `network private-endpoint-connection delete`?

Reply:
Sure